velox/docs/functions/presto/binary.rst

```diff
@@ -130,3 +130,19 @@ Binary Functions
 .. function:: xxhash64(binary) -> varbinary
 
     Computes the xxhash64 hash of ``binary``.
+    
+.. function:: lpad(binary, padbinary, size) -> varbinary
+    :noindex:
+    
+    Left pads ``binary`` to ``size`` bytes with ``padbinary``.
+    If ``size`` is less than the length of ``binary``, the result is
+    truncated to ``size`` characters. ``size`` must not be negative
+    and ``padbinary`` must be non-empty.
+
+.. function:: rpad(binary, padbinary, size) -> varbinary
```

Comment:
Is there a limit on 'size'? I remember we saw crashes without the limit. It would be nice to document that limit.

Reply:
there's a hard coded limit on 'size' which is set to 1 MiB. Ive updated the comments here to reflect the same.